internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -463,6 +484,117 @@ func convertEnvSecurityGroupCfg(mft *manifest.Environment) (*template.SecurityGr
 	}, nil
 }
 
+func (s *LoadBalancedWebService) convertApplicationLoadBalancer() (*template.ALBListener, error) {
```

Comment:
Should we rename this function as it seems to return `ALBListener`. `convertApplicationLoadBalancer` I would assume it returns ALB config but it actually returns listener config.